src/Libraries/Microsoft.Extensions.Resilience/Resilience/Internal/ResilienceTagNames.cs

```diff
@@ -5,13 +5,13 @@ namespace Microsoft.Extensions.Resilience.Internal;
 
 internal static class ResilienceTagNames
 {
-    public const string FailureSource = "failure-source";
+    public const string FailureSource = "resilience.failure.source";
 
-    public const string FailureReason = "failure-reason";
+    public const string FailureReason = "resilience.failure.reason";
 
-    public const string FailureSummary = "failure-summary";
+    public const string FailureSummary = "resilience.failure.summary";
```

Comment:
what's the expected cardinality of this attribute? It looks a bit scary - `_exceptionSummarizer.Summarize(e).ToString()`